doc/flame/examples/lib/ray_trace.dart

```diff
@@ -10,7 +10,7 @@ import 'package:flutter/material.dart';
 
 class RayTraceExample extends FlameGame
     with HasCollisionDetection, TapDetector {
-  Paint paint = BasicPalette.red.withOpacity(0.6).paint();
+  Paint paint = Paint()..color = Colors.red.withOpacity(0.6);
```

Comment:
```suggestion
  final Paint paint = Paint()..color = Colors.red.withOpacity(0.6);
```

Reply:
this is copied as is from main - you can fix on a separate PR!